eng/versioning/pom_file_version_scanner.ps1

```diff
@@ -150,7 +154,14 @@ function Get-ArtifactsList-Per-Service-Directory {
 
     # Get all of the yml files under sdk/
     $ymlFiles = Get-ChildItem -Path $SdkRoot -Recurse -Depth 3 -File -Filter "ci*yml"
+    if ($ymlFiles.Length -eq 0) {
```

Comment:
TIL I was pretty sure this value would be null, and it is, but apparently `$null.Length` works in powershell...